chart/templates/check-values.yaml

```diff
@@ -21,7 +21,18 @@ The sole purpose of this yaml file is it to check the values file is consistent
 
 {{- /*
 ##############################
-   Redis related checks
+  Executors related checks
+#############################
+*/ -}}
+  {{- $supportedExecutors := list "LocalExecutor" "CeleryExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
+
+  {{- if not (has .Values.executor $supportedExecutors) }}
+    {{ required "Unsupported executor. Supported values: LocalExecutor, CeleryExecutor, KubernetesExecutor, CeleryKubernetesExecutor." nil }}
```

Comment:
Can we reuse the variable `supportedExecutors`? So that avoid hard-coding the same thing.

Reply:
Oh The change in this file should be deleted because I was able to validate it with JSON Schema.